tests/unit/conftest.py

```diff
@@ -4,17 +4,22 @@
 # pylint: disable=duplicate-code,missing-function-docstring
 
 """Unit test fixtures."""
+
+import pathlib
 import typing
 from unittest.mock import patch
 
 import ops.testing
 import pytest
 
-from src.charm import ChronyCharm
+import keychain
+from charm import ChronyCharm
 
 
 @pytest.fixture
-def harness() -> typing.Generator[ops.testing.Harness, None, None]:
+def harness(  # pylint: disable=too-many-locals
```

Comment:
Normally this should be accompanied by an explanatory comment.

Reply:
The comment saying 'too many local variables' might be a little bit redundant here, which is why I omitted it.